assemblyline_core/dispatching/schedules.py

```diff
@@ -69,7 +70,11 @@ def build_schedule(self, submission: Submission, file_type: str, file_depth: int
             accepted = not service.accepts or re.match(service.accepts, file_type)
             rejected = bool(service.rejects) and re.match(service.rejects, file_type)
 
-            if accepted and not rejected:
+            # Ensure that the submission's submitter is allowed to use this service before adding to schedule
```

Comment:
I'm worried this will significantly slow down scheduling. We should do a little bit of caching.